src/PhotosphereViewer.tsx

```diff
@@ -213,6 +214,7 @@ function PhotosphereViewer({
   onChangePS,
   onViewerClick,
   onUpdateHotspot,
+  photosphereOptions = [],
```

Comment:
I'd recommend removing the default value `= []` here since `photosphereOptions` is also optional in `PopOver`.